### PR TITLE
TEMPLATE: Keep pipeline name in version.yml file

### PR DESCRIPTION
**Pull request by maxulysse**
Why going for `pipeline` when we have shortname?

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

File: nf_core/pipeline-template/workflows/pipeline.nf [hunk 1]
```diff
@@ -47,7 +47,7 @@ workflow {{ short_name|upper }} {
     softwareVersionsToYAML(ch_versions)
         .collectFile(
             storeDir: "${params.outdir}/pipeline_info",
-            name: {% if is_nfcore %}'nf_core_' {% else %} '' {% endif %} + 'pipeline_software_' + {% if multiqc %} 'mqc_' {% else %} '' {% endif %} + 'versions.yml',
+            name: {% if is_nfcore %}'nf_core_' {% else %} '' {% endif %} + '{{ short_name }}_software_' + {% if multiqc %} 'mqc_' {% else %} '' {% endif %} + 'versions.yml',
```
Comment (maxulysse) [inline nf_core/pipeline-template/workflows/pipeline.nf:50]:
```suggestion
            name: {% if is_nfcore %}'nf_core_'  + {% else %}{% endif %} '{{ short_name }}_software_' + {% if multiqc %} 'mqc_' {% else %} '' {% endif %} + 'versions.yml',
```

**Comment by mashehu** [comment]
@nf-core-bot changelog

File: nf_core/pipeline-template/workflows/pipeline.nf [hunk 2]
```diff
@@ -47,7 +47,7 @@ workflow {{ short_name|upper }} {
     softwareVersionsToYAML(ch_versions)
         .collectFile(
             storeDir: "${params.outdir}/pipeline_info",
-            name: {% if is_nfcore %}'nf_core_' {% else %} '' {% endif %} + 'pipeline_software_' + {% if multiqc %} 'mqc_' {% else %} '' {% endif %} + 'versions.yml',
+            name: {% if is_nfcore %}'nf_core_'  + {% else %}{% endif %} '{{ short_name }}_software_' {% if multiqc %} + 'mqc_' {% else %}{% endif %} + 'versions.yml',
```
Comment (mirpedrol) [inline nf_core/pipeline-template/workflows/pipeline.nf:50]:
```suggestion
            name: {% if is_nfcore %}'nf_core_'  + {% endif %} '{{ short_name }}_software_' {% if multiqc %} + 'mqc_' {% endif %} + 'versions.yml',
```

**Review by mirpedrol — APPROVED** [review]
LGTM 🙂

**Pull request merged**